src/index.js

```diff
@@ -27,7 +27,7 @@ export default async function register(app) {
           [{ name: 1 }, { name: "c2_name" }],
           [{ relatedTagIds: 1 }, { name: "c2_relatedTagIds" }],
           [{ shopId: 1 }, { name: "c2_shopId" }],
-          [{ slug: 1 }, { unique: true }]
```

Comment:
Do we need a migration to drop the old index?

Reply:
yes, but it will need to be done in 
https://github.com/reactioncommerce/api-plugin-authorization-simple ?

Reply:
I'm sorry I'm not exactly sure what you mean, but the migration would have nothing do with` api-plugin-authorization-simple` specificallly.

You're right however that `api-plugin-authorization-simple` is one of the two plugins currently using migrations.
A migration in a similar form as done there would need to be added to this tags repo and then imported here:
https://github.com/reactioncommerce/api-migrations.